File: librarymanagementsystem-backend/src/librarymanagementsystembackend/Application/Features/MaterialCopies/Commands/Update/UpdatedMaterialCopyResponse.cs
using NArchitecture.Core.Application.Responses;

namespace Application.Features.MaterialCopies.Commands.Update;

public class UpdatedMaterialCopyResponse : IResponse
{
    public Guid Id { get; set; }
    public DateTime DateReceipt { get; set; }
    public string Status { get; set; }
    public bool IsReserved { get; set; }
    public bool IsReservable { get; set; }
    public Guid MaterialId { get; set; }
    public Guid BranchId { get; set; }
    public Guid LocationId { get; set; }
}

File: librarymanagementsystem-frontend/src/app/features/models/responses/material-copies/updated-material-copy-response.ts
export interface UpdatedMaterialCopyResponse {
	id: string;
	dateReceipt: string;
	status: string;
	isReserved: boolean;
	isReservable: boolean;
	materialId: string;
	branchId: string;
	locationId: string;
    updatedDate: Date;
}